src/app/security-group/sg-rules/sg-rules.component.html

```diff
@@ -63,7 +63,7 @@
                 <input
                   matInput
                   maxValue="255"
-                  [minValue]="cidrIpVersion === IPVersion.ipv6 ? 0 : -1"
+                  minValue="0"
```

Comment:
It should be possible to use ICMP type -1 for IPv4 rules. The same for ICMP codes.

Reply:
Please provide link that ICMP Type can have "-1" value. I can't find that info on the internet.
https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml